packages/amplication-build-manager/src/queue/queue.service.ts

```diff
@@ -1,16 +1,29 @@
-import { Inject, Injectable } from "@nestjs/common";
+import { Inject, Injectable, OnModuleInit } from "@nestjs/common";
 import { ClientKafka } from "@nestjs/microservices";
 
 export const KAFKA_CLIENT = "KAFKA_CLIENT";
 
 @Injectable()
-export class QueueService {
+export class QueueService implements OnModuleInit {
   constructor(
     @Inject(KAFKA_CLIENT)
     private readonly kafkaClient: ClientKafka
   ) {}
 
-  emitMessage(topic: string, message: string): void {
-    this.kafkaClient.emit(topic, message);
+  async onModuleInit() {
+    await this.kafkaClient.connect();
+  }
```

Comment:
The problem with this solution is that it would break the current build process that requires to start the nestjs application to generate graphql schemas. The nestjs won't be able to start unless a running kafka cluster is available.

Building process should be independent from other systems like kafka

Reply:
I removed connect call.